internal/cli/atlas/livemigrations/cutover.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livemigrations
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var createTemplate = "Live migration '{{.ID}}' successfully started.\n"
```

Comment:
cutover is not to start a migration but to finish it

Reply:
ah okay, the doc says "starts the migration of one deployment" so I was confused
![Screenshot 2021-09-14 at 13 31 06](https://user-images.githubusercontent.com/5663078/133257936-83a8507b-4fd1-4c87-a1c1-d1da4d57877f.png)

Reply:
can you please open a docs ticket? or/and review the OpenAPI docs for that endpoint to make sure is correct?

Reply:
https://jira.mongodb.org/browse/DOCSP-18677

Reply:
Looks like the original OpenAPI text was not used: https://github.com/10gen/mms/blob/master/server/src/main/com/xgen/module/liveimport/api/res/ApiLiveMigrationsResource.java#L161-L163